prism/1.explore_prism.ipynb

```diff
@@ -0,0 +1,1310 @@
+       "ENGINEERED                     4\n",
+       "SALIVARY_GLAND                 2\n",
+       "FIBROBLAST                     1\n",
+       "UPPER_AERODIGESTIVE            1\n",
+       "Name: tissue, dtype: int64"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# How many tissues in secondary screen\n",
+    "prism_cell_df.loc[:, \"tissue\"].value_counts()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "f7d69b20-8cca-4fd6-8022-06741e4cba7a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "1448\n",
+      "530\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "EGFR inhibitor                      376\n",
+       "HDAC inhibitor                      240\n",
+       "topoisomerase inhibitor             192\n",
+       "glucocorticoid receptor agonist     192\n",
+       "CDK inhibitor                       192\n",
+       "tubulin polymerization inhibitor    184\n",
+       "MEK inhibitor                       184\n",
+       "mTOR inhibitor                      160\n",
+       "Aurora kinase inhibitor             152\n",
+       "histamine receptor antagonist       144\n",
+       "HSP inhibitor                       144\n",
+       "PI3K inhibitor                      144\n",
+       "protein synthesis inhibitor         144\n",
+       "adrenergic receptor antagonist      136\n",
+       "RAF inhibitor                       128\n",
+       "cyclooxygenase inhibitor            120\n",
+       "glutamate receptor antagonist       112\n",
+       "PARP inhibitor                      112\n",
+       "adrenergic receptor agonist         112\n",
+       "dopamine receptor antagonist        104\n",
+       "Name: moa, dtype: int64"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# How many unique treatments\n",
+    "print(prism_trt_df.name.nunique())\n",
+    "\n",
+    "# How many MOAs\n",
+    "print(prism_trt_df.moa.nunique())\n",
+    "prism_trt_df.moa.value_counts().head(20)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "624a8822-2365-4a1e-9f86-37080ecb42f0",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "8     1279\n",
+       "16     166\n",
+       "48       2\n",
+       "24       1\n",
+       "Name: dose, dtype: int64"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# How many doses per compound\n",
+    "prism_trt_df.groupby(\"name\").dose.count().value_counts()"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
```

Comment:
Is this environment / kernel accurate for this notebook? If not, consider rerunning in the correct environment.

Reply:
nope! will rerun, thanks!